### DISCUSSION TITLE
I save workflow on flowise hosted on Azure but disappears upon inactivity

### DISCUSSION THREAD

**Discussion by Prosysly** [FlowiseAI/Flowise#604] [Q&A]
I save workflow on flowise hosted on Azure but disappears upon inactivity. 

I am able to execute and test chats, all works fine. I am able to save workflow but it won't stay saved on the Flowise AI dashboard.

Thanks in advance.

AG

**Comment by HenryHengZJ** [comment]
you may need to add persistent storage - https://docs.flowiseai.com/deployment/azure#create-a-container-instance-with-persistent-storage

**Reply by twentytwokhz** [reply]
Personally I tried the persistant storage option  and it did not work for me.
The `/opt/flowise/.flowise` path seems to not be created inside the file share. Ideas?

**Reply by AutoAlchemists** [reply]
same issue, except im hosting on Pinecone

**Reply by cloudberry-py** [reply]
try adding env var DATABASE_PATH=/opt/flowise/.flowise and also volume for azurefile mount path to /opt/flowise/.flowise
